pyallsky/imagecapture.py

```diff
@@ -40,6 +42,11 @@ def capture_image_device(device, exposure, dark=False):
     logging.info('Connecting to camera')
     cam = AllSkyCamera(device)
 
+    if heating:
```

Comment:
like you did above, could do this with the ternary operator

`cam.activate_heater() if heating else cam.deactivate_heater()`